src/ocean/util/app/ext/ReopenableFilesExt.d

```diff
@@ -130,6 +224,37 @@ public class ReopenableFilesExt : IApplicationExtension, ISignalExtExtension
         }
     }
 
+    /****************************************************************************
+
+        Reopen command to trigger from the Unix Domain socket. It reads
+        the log name to reload and reloads the particular log file.
+
+        Params:
+            args = list of arguments received from the socket - should contain
+                the filename of the log to rotate.
+            send_response = delegate to send the response to the client
+
+    *****************************************************************************/
+
+    private void socketReloadCommand ( cstring[] args,
+            void delegate ( cstring response ) send_response )
+    {
+        if (args.length == 0)
+        {
+            send_response("ERROR: missing name of the file to rotate.\n");
+            return;
+        }
+
+        auto filename = args[0];
```

Comment:
Could we use a `while` loop here, so one can pass multiple files ?

Reply:
I didn't anticipated that one can do that, since logrotate doesn't do it, but it makes sense, I'll update it.